python/pyspark/mllib/recommendation.py

```diff
@@ -115,11 +128,39 @@ def predictAll(self, user_product):
         return self.call("predict", user_product)
 
     def userFeatures(self):
+        """
+        Returns a coupled RDD, where the first element is the user and the
+        second is an array of features corresponding to that user.
+        """
         return self.call("getUserFeatures").mapValues(lambda v: array.array('d', v))
 
     def productFeatures(self):
+        """
+        Returns a coupled RDD, where the first element is the product and the
+        second is an array of features corresponding to that product.
+        """
         return self.call("getProductFeatures").mapValues(lambda v: array.array('d', v))
 
+    def recommendUsers(self, product, num):
+        """
+        Recommends the top "num" number of products for a given user.
+        This is done by returning a tuple of Rating objects, the second id (product)
+        being constant and sorted according to the rating.
```

Comment:
`Recommends the top "num" number of products for a given user and returns a list or Rating objects sorted by the predicted rating in descending order.`